check-diff.sh

```diff
@@ -69,10 +70,16 @@ function set_environment_variables {
 			DIFF_BASE=${DIFF_BASE:-$TRAVIS_COMMIT^}
 		fi
 		DIFF_HEAD=${DIFF_HEAD:-$TRAVIS_COMMIT}
+	fi
+
+	if [[ -n "${BITBUCKET_BRANCH+set}" ]]; then
```

Comment:
The `[[ -n "${BITBUCKET_BRANCH+set}" ]]` statement will seem to work because it will _always_ work. By that I mean, it will always be true, since `"+set"` is not null.